src/app/app.component.ts

```diff
@@ -2,29 +2,42 @@
  * Angular 2 decorators and services
  */
 import { Component, ViewEncapsulation } from '@angular/core';
+import { cloneDeep } from 'lodash';
+import { ActivatedRoute } from '@angular/router';
+import { AuthenticationService, Broadcaster } from 'ngx-login-client';
+import { Notification, NotificationActionCallback } from 'ngx-fabric8-wit';
```

Comment:
Also, the callbacks thing. Without that there is no way for a decoupled producer of notifications to react to actions that I can see.